src/algebra/algebra/spectrum.lean

```diff
@@ -212,6 +213,26 @@ theorem preimage_units_mul_eq_swap_mul {a b : A} :
   (coe : Rˣ → R) ⁻¹' σ (a * b) = coe ⁻¹'  σ (b * a) :=
 by { ext, exact unit_mem_mul_iff_mem_swap_mul, }
 
+section star
+
+variables [star_add_monoid R] [star_ring A] [star_module R A]
+
+local postfix `⋆`:std.prec.max_plus := star
```

Comment:
I don't think this makes things more readable: it doesn't really affect the length much, but it's a little jarring to the reader because we don't do this elsewhere.

Reply:
Point taken here, but just FYI, we are starting to use some form of (local) notation for the star operation in `analysis.normed_space.star.*`